src/sympc/__init__.py

```diff
@@ -25,6 +25,9 @@
 try:
     # third party
     import syft
+    from syft.experimental_flags import flags
+
+    flags.APACHE_ARROW_TENSOR_SERDE = False
```

Comment:
Q: This is because syft uses `ARROW` for numpy?

Reply:
Crct george,Syft used `ARROW` for numpy. The `serde` is buggy in the case of `ARROW` ,
The de-serialization of boolean type are erroneous.
I opened an issue regarding the same in PySyft george now.
https://github.com/OpenMined/PySyft/issues/5930